### ISSUE TITLE
Starved to death on Tyder and got spammed with starvation messages until I and everyone near me crashed

### ISSUE THREAD

**Issue by bluelimeminer** [StarQuestMinecraft/StarQuestPublic#1158]
![capture](https://cloud.githubusercontent.com/assets/9390178/17090768/a1fd799c-51e8-11e6-89df-91adf7af8bea.PNG)

**Comment by CrazyCreeper1o1** [comment]
Alright, I just suicided on Erilon losing a few stacks of cobble, but the same thing happened and I think I can add some insight to the issue. I don't think the reason people are crashing is directly because of the deaths. After I jumped off a building, I got spammed but eventually I spawned in my cryopod a little bit aways from the point I died. Then, as I approached, I noticed that there was a LOT more cobble than I had. I think people crashed because they were trying to load several thousand stacks of entities at the same time. I also got some screenshots of the crash report after rebooting minecraft saying that item entity 153435 has no item right before I started dying. I hope this helps!
![screenshot 6](https://cloud.githubusercontent.com/assets/19254135/17097975/f25118b4-5214-11e6-9ab1-0810dc890de2.png)
![screenshot 5](https://cloud.githubusercontent.com/assets/19254135/17097976/f3b2c43c-5214-11e6-94e9-8d1768c69b75.png)

Edit: Wait, it happened again and this time the item entity was 238631. Also, this time I lost 17 dispensers and a modded tier 3 pickaxe in addition to the worthless items...
